pythonforandroid/bootstraps/common/build/build.py

```diff
@@ -861,6 +863,14 @@ def parse_args_and_make_package(args=None):
                     help='Use that parameter if you need to implement your own PythonServive Java class')
     ap.add_argument('--activity-class-name', dest='activity_class_name', default=DEFAULT_PYTHON_ACTIVITY_JAVA_CLASS,
                     help='The full java class name of the main activity')
+    ap.add_argument('--network-security-config', dest='network_security_config', default='',
+                    help='Add a Network Security Configuration file path to AndroidManifest.xml')
+    ap.add_argument('--uses-cleartext-traffic', dest='uses_cleartext_traffic', default='',
+                    help='Indicate that app intends to use cleartext network traffic in AndroidManifest.xml')
+    ap.add_argument('--service-class-name', dest='service_class_name', default='org.kivy.android.PythonService',
+                    help='Use that parameter if you need to implement your own PythonServive Java class')
```

Comment:
We already have a `--service-class-name` (See above)

Reply:
:+1: